app/views/policies/_basic_policy_details.slim

```diff
@@ -4,31 +4,31 @@
      | Category
      span.required *
     .col-md-6
-      = p.select :company_category_id, options_for_select(Category.all.map{ |c| [c.name, c.id]}), {prompt: ApplicationHelper::DEFAULT_SELECT_OPTION}, {class: "form-control select"}
+      = p.select :company_categories_id, options_for_select(Category.all.map{ |c| [c.name, c.id]},selected: @category_id), {prompt: ApplicationHelper::DEFAULT_SELECT_OPTION}, {class: "form-control select"}
 = f.fields_for :CategoryCompany do |p|
   .form-group
     label.control-label.col-md-3
       | Company name
       span.required *
     .col-md-6
-      = p.select :company_id, options_for_select(Company.all.map{ |c| [c.name, c.id]}), {prompt: ApplicationHelper::DEFAULT_SELECT_OPTION}, {class: "form-control select"}
+      = p.select :company_id, options_for_select(Company.all.map{ |c| [c.name, c.id]},selected: @category_id), {prompt: ApplicationHelper::DEFAULT_SELECT_OPTION}, {class: "form-control select"}
```

Comment:
@GauravMakwana-BTC : Is it correct that you are using same category id for Company & category?